Dockerfile

```diff
@@ -42,6 +42,13 @@ COPY --chown=appuser:appuser . .
 # Build application
 RUN yarn build
 
+# Clean all depencies...
+RUN rm -rf node_modules
+RUN yarn cache clean
+
+# ... and install only production dependencies
+RUN yarn install --production
```

Comment:
`npm` has a feature where deps can be pruned to match a target. `yarn` does not have it. Instead we have to reinstall. I adopted this approach so that if we transition into using `npm` due to the major changes in `yarn2`, we can adopt the `prune` patterns and save some time in the image creation.